packages/block-editor/src/components/block-popover/inbetween.js

```diff
@@ -106,16 +106,7 @@ function BlockPopoverInbetween( {
 						nextRect && previousRect
 							? nextRect.top - previousRect.bottom
 							: 0;
-
-					if ( isRTL() ) {
-						// vertical, rtl
-						left = previousRect
-							? previousRect.right
-							: nextRect.right;
-					} else {
-						// vertical, ltr
-						left = previousRect ? previousRect.left : nextRect.left;
-					}
+					left = previousRect ? previousRect.left : nextRect.left;
```

Comment:
Out of curiosity, why was the `isRTL()` check here unnecessary?

Reply:
just imagine this visually. 

In LTR languages, the previous element is on the left of the next element but in RTL it's on the right. meaning that if we want to position a "div" between the two elements, its "left" coordinate should be the "right" coordinate of the "previous" element in LTR but it should be the "left" coordinate of the "next" element in RTL.

Reply:
Oh I guess the question here is the "opposite". It's unnecessary here because in the case of the "vertical" in-between inserter, the "left" position is the same for both blocks anyway, so the left position of the empty space between them is also the same "left" position, so either prev.left or next.left. (and shouldn't be right in any case)

Reply:
Thank you for the explanation — it makes sense to remove the RTL check for the vertical inserter :)